install/syndesis.yml

```diff
@@ -802,7 +706,41 @@ objects:
             name: syndesis-sampledb-config
           - mountPath: /opt/app-root/src/postgresql-cfg/
             name: syndesis-db-conf
+        - capabilities: {}
+          env:
+          - name: DATA_SOURCE_NAME
+            value: postgresql://${POSTGRESQL_USER}:${POSTGRESQL_PASSWORD}@syndesis-db:5432/syndesis?sslmode=disable
```

Comment:
Now that the exporter is running in the same pod as the DB, we can connect via `localhost`, right?